tests/test_ecdsa.py

```diff
@@ -50,3 +50,8 @@ def test_ecdsa_sign_without_r_value_grinding(self):
             sigencode=ecc.ecdsa_der_sig_from_r_and_s, grind_r_value=False,
         )
         self.assertEqual('3045022100902a288b98392254cd23c0e9a49ac6d7920f171b8249a48e484b998f1874a2010220723d844826828f092cf400cb210c4fa0b8cd1b9d1a7f21590e78e022ff6476b9', sig1.hex())
+
+    def test_ecdsa_verify_with_unusual_sig64_len(self):
```

Comment:
thx, I've manually merged your testcase in https://github.com/spesmilo/electrum-ecc/commit/19c1475f486944cea29014cd6ebd371918d928b1